tests/platforms/web/static/test_build.py

```diff
@@ -168,27 +224,40 @@ def mock_run(*args, **kwargs):
         )
 
 
-def test_build_app_custom_pyscript_toml(build_command, first_app_generated, tmp_path):
-    """An app with extra pyscript.toml content can be written."""
+def test_build_app_no_config(build_command, first_app_generated, tmp_path):
+    """An app with no config.toml supplied by a wheel gets a basic config."""
+
     bundle_path = tmp_path / "base_path/build/first-app/web/static"
 
-    first_app_generated.extra_pyscript_toml_content = (
-        "\n".join(
-            [
-                # A custom top level key
-                'something = "custom"',
-                # A key overwriting something in the main namespace
-                'packages = ["something-custom"]',
-                # A table overwriting an existing value
-                "[splashscreen]",
-                "wiggle = false",
-                # A custom array of tables
-                "[[runtimes]]",
-                'src = "https://example.com/pyodide.js"',
-            ]
-        )
-        + "\n"
-    )
+    # Mock some wheels without a config.toml
+    def mock_run(*args, **kwargs):
```

Comment:
This looks like it has been significantly reproduced in multiple tests; it looks like there's an opportunity to refactor so there's a common utility method.